docs(react-native): Revamp integrations page with categories and missing integrations (#17801)
getsentry/sentry-docs: docs/platforms/react-native/integrations/deeplink.mdx

---
title: Deep Link
description: "Capture deep link navigation as breadcrumbs in your React Native application."
sidebar_order: 100
---

> _(Available in version 8.8.0 and above)_

Author: Version numbers are correct — verified from the SDK repo:

- **deeplink**: `8.8.0` is the first tag containing the commit that added `deeplinkIntegration` (`git tag --contains 7ac33789`), also present in the CHANGELOG.
- **spotlight**: `6.0.0` is the first tag containing the commit that added `spotlightIntegration` (`git tag --contains ddcabffc`).

No change needed here.

_Import name: `Sentry.deeplinkIntegration`_

The `deeplinkIntegration` automatically captures breadcrumbs when your app receives deep links via React Native's `Linking` API. It tracks both cold start URLs (the link that opened the app) and warm open URLs (links received while the app is running).

> ## Setup

Author: Good point — addressed in e3e26ba. Added an Alert noting that the app must have deep linking configured natively (URL schemes or universal links) for the integration to receive URLs, with a link to React Native's Linking docs.

---
title: Deep Link
description: "Capture deep link navigation as breadcrumbs in your React Native application."
sidebar_order: 100
---

_(Available in version 8.8.0 and above)_

_Import name: `Sentry.deeplinkIntegration`_

The `deeplinkIntegration` automatically captures breadcrumbs when your app receives deep links via React Native's [`Linking`](https://reactnative.dev/docs/linking) API. It tracks both cold start URLs (the link that opened the app) and warm open URLs (links received while the app is running).

<Alert level="info" title="Note">

> This integration relies on React Native's `Linking` module. Your app must have deep linking configured natively (URL schemes or universal links) for the integration to receive URLs.

Reviewer: ```suggestion
This integration relies on React Native's `Linking` module. To receive URLs, your app must have native deep linking configured using URL schemes or universal links.
```

Author: Thank you for you suggestions 🙇
